packages/nc-gui/components/smartsheet/Row.vue

```diff
@@ -37,9 +37,9 @@ const reloadViewDataTrigger = inject(ReloadViewDataHookInj)!
 // override reload trigger and use it to reload row
 const reloadHook = createEventHook()
```

Comment:
You could update this `createEventHook<boolean | void>()`

Then you could remove the typecast below, i.e.
`reloadHook.on(shouldShowLoading => {})`